src/resources/parser/json-model.js

```diff
@@ -542,7 +542,7 @@ pc.extend(pc, function () {
                 var vertexData = modelData.vertices[i];
 
                 // Check to see if we need to generate tangents
-                if (vertexData.position && vertexData.normal && vertexData.texCoord0) {
+                if (!vertexData.tangent && vertexData.position && vertexData.normal && vertexData.texCoord0) {
```

Comment:
The PlayCanvas model JSON writer never writes tangent data which explains this line. Are you creating your own model JSON with this data?

Reply:
I am writing a THREE.js to PlayCanvas parser. 😃 The THREE.js geometry contains tangents data. What's more, calculating tangents in client/browser can be slow if the model is complex.

Reply:
Fair enough. I'll merge then. 😄

Reply:
Fair enough